portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,21 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact about Fatima to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const fatimaFacts =
+      ['I am left handed!', 'I have an older brother named Humza!', 
+      'My favorite color is torquise!', 'I love photography!',
```

Comment:
turquoise?

Reply:
oops i will definitely run things through Grammarly next time—thanks for the catch!